.github/workflows/ci.yml

```diff
@@ -15,4 +15,26 @@ jobs:
       - name: Install
         run: npm ci
       - name: Lint
-        run: npm run lint
\ No newline at end of file
+        run: npm run lint
+
+  postman:
+    runs-on: ubuntu-latest
+    # With the exception of GITHUB_TOKEN, secrets are not passed to the runner
+    # when a workflow is triggered from a forked repository.
+    if: github.repository == github.event.repository.full_name
+    steps:
+      - name: Check out git repository
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: npm ci
+        working-directory: packages/tests-postman
+      - name: Run Postman tests
+        run: node .
```

Comment:
nit: this should be `npm run test:all`

also any specific reason we're cding into the packages file and not just running it from base with lerna?

Reply:
Nope, just my lack of knowledge on using lerna :) Looking into fixing this now.

Reply:
ok, updated scripts to use lerna.